src/include/job.h

```diff
@@ -552,43 +552,39 @@ struct job {
 	char qs_hash[DIGEST_LENGTH];
 #endif
 	struct jobfix {
-		int	    ji_jsversion;	/* job structure version - JSVERSION */
-		int	    ji_state;		/* internal copy of state */
-		int	    ji_substate;	/* job sub-state */
-		int	    ji_svrflags;	/* server flags */
-		int	    ji_numattr;		/* not used */
-		int	    ji_ordering;	/* special scheduling ordering */
-		int	    ji_priority;	/* internal priority */
-		time_t  ji_stime;		/* time job started execution */
-		time_t  ji_endtBdry;	/* estimate upper bound on end time */
-
-		char    ji_jobid[PBS_MAXSVRJOBID+1];   /* job identifier */
-		char    ji_fileprefix[PBS_JOBBASE+1];  /* no longer used */
-		char    ji_queue[PBS_MAXQUEUENAME+1];  /* name of current queue */
-		char    ji_destin[PBS_MAXROUTEDEST+1]; /* dest from qmove/route */
+		int	ji_jsversion;				/* job structure version - JSVERSION */
+		int	ji_state;				/* internal copy of state */
+		int	ji_substate;				/* job sub-state */
+		int	ji_svrflags;				/* server flags */
+		time_t  ji_stime;				/* time job started execution */
+
+		char	ji_jobid[PBS_MAXSVRJOBID+1];		/* job identifier */
+		char	ji_fileprefix[PBS_JOBBASE+1];		/* no longer used */
```

Comment:
comment says no longer used? so can we get rid of it?

Reply:
If we are going to change/modify job_fix or task_fix structure of job then i guess need to add corresponding changes in pbs_upgrade_job tool to support overlay upgrade feature

Reply:
I have another PR in works to refactor job's quick save. I can consider this one in that PR. There is still code using this variable. May be we dont get to that code path anymore.